quarkus-maven-plugin not working correctly for project creation.

**Describe the bug**
quarkus-maven-plugin not working correctly for project creation.

**Expected behavior**
working without problem to locate gradle wrapper. 1.4.x is working.

**Actual behavior**
```
D:\Quarkus\templates>mvn io.quarkus:quarkus-maven-plugin:1.5.1.Final:create -DbuildTool=gradle -DoutputDirectory=1.5.1.Final -DplatformVersion=1.5.1.Final
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- quarkus-maven-plugin:1.5.1.Final:create (default-cli) @ standalone-pom ---
Set the project groupId [org.acme.quarkus.sample]:
Set the project artifactId [my-quarkus-project]:
Set the project version [1.0-SNAPSHOT]:
Do you want to create a REST resource? (y/n) [no]:
Creating a new project in D:\Quarkus\templates\1.5.1.Final\my-quarkus-project
[ERROR] Unable to copy Gradle wrapper from platform descriptor
java.io.IOException: Failed to locate gradle-wrapper\gradlew on the classpath
    at io.quarkus.platform.descriptor.loader.json.ClassPathResourceLoader.loadResource (ClassPathResourceLoader.java:23)
    at io.quarkus.platform.descriptor.loader.json.impl.QuarkusJsonPlatformDescriptor.loadResource (QuarkusJsonPlatformDescriptor.java:126)
    at io.quarkus.maven.CreateProjectMojo.createGradleWrapper (CreateProjectMojo.java:245)
    at io.quarkus.maven.CreateProjectMojo.execute (CreateProjectMojo.java:226)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO]
[INFO] ========================================================================================
[INFO] Your new application has been created in D:\Quarkus\templates\1.5.1.Final\my-quarkus-project
[INFO] Navigate into this directory and launch your application with mvn quarkus:dev
[INFO] Your application will be accessible on http://localhost:8080
[INFO] ========================================================================================
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.213 s
[INFO] Finished at: 2020-06-15T17:04:15+08:00
[INFO] ------------------------------------------------------------------------
```

**To Reproduce**
Steps to reproduce the behavior:
1. Type the command "mvn io.quarkus:quarkus-maven-plugin:1.5.1.Final:create -DbuildTool=gradle -DoutputDirectory=1.5.1.Final -DplatformVersion=1.5.1.Final" on windows console.
2. You will see the problem.

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Microsoft Windows [版本 10.0.18363.900]
 - Output of `java -version`: 
openjdk version "11.0.6" 2020-01-14
OpenJDK Runtime Environment GraalVM CE 20.0.0 (build 11.0.6+9-jvmci-20.0-b02)
OpenJDK 64-Bit Server VM GraalVM CE 20.0.0 (build 11.0.6+9-jvmci-20.0-b02, mixed mode, sharing)
 - Quarkus version or git rev: 1.5.1.Final
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)

--- a/devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java
+++ b/devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java
@@ -16,7 +16,6 @@ import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Locale;
@@ -220,7 +219,7 @@ public class CreateProjectMojo extends AbstractMojo {
             Files.createDirectories(projectDirPath.resolve("gradle/wrapper"));
 
             for (String filename : CreateUtils.GRADLE_WRAPPER_FILES) {
-                byte[] fileContent = platform.loadResource(Paths.get(CreateUtils.GRADLE_WRAPPER_PATH, filename).toString(),
+                byte[] fileContent = platform.loadResource(CreateUtils.GRADLE_WRAPPER_PATH + '/' + filename,
                         is -> {
                             byte[] buffer = new byte[is.available()];
                             is.read(buffer);

